docs/ossm/ambient/observability.md

```diff
@@ -0,0 +1,366 @@
+| waypoint proxies | generates L7 telemetry for HTTP, HTTP/2, gRPC traffic metrics and distributed traces. |
+
+Red Hat OpenShift Service Mesh integrates with the following Red Hat OpenShift Observability components in ambient mode:
+
+- OpenShift Cluster Monitoring components
+- Red Hat OpenShift Distributed tracing platform
+
+OpenShift Service Mesh also integrates with:
+
+- Kiali provided by Red Hat, a powerful console for visualizing and managing your service mesh.
+- OpenShift Service Mesh Console (OSSMC) plugin, an OpenShift Container Platform console plugin that seamlessly integrates Kiali console features into your OpenShift console.
+
+## Configuring Metrics with OpenShift Cluster Monitoring and Service Mesh Ambient mode
+
+Monitoring stack components are deployed by default in every OpenShift Container Platform installation. These components include Prometheus, Alertmanager, Thanos Querier, and others.
+
+When you have enrolled your application in the Service Mesh Ambient mesh mode, you can monitor the [Istio Standard Metrics](https://istio.io/latest/docs/reference/config/metrics/) of your application from the ztunnel resource and the waypoint proxies. The ztunnel also exposes [a variety of DNS and debugging metrics](https://github.com/istio/ztunnel/blob/master/README.md#metrics).
+
+Because the Service Mesh Ambient mode has two levels of proxies, there are two sets of metrics that can be collected for each application service. The Layer 4 TCP related metrics can be collected from the ztunnel resource and the waypoint proxies. The Layer 7 metrics such as HTTP traffic metrics can be collected from the waypoint proxies.
+
+### Prerequisites
+
+1. Red Hat OpenShift Service Mesh 3 Ambient mode Istio, IstioCNI and ZTunnel CRs are created. If you have not already done so, install the OpenShift Service Mesh Operator along with Istio in ambient mode using the following [steps](https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ambient/README.md#3-procedure-to-install-istios-ambient-mode).
+
+2. OpenShift Cluster Monitoring User-workload monitoring is enabled. You can enable that by applying the following ConfigMap change.
+
+  This ConfigMap change is the only command you need from the doc page [Enabling monitoring for user-defined projects](https://docs.redhat.com/en/documentation/openshift_container_platform/4.19/html/monitoring/configuring-user-workload-monitoring#enabling-monitoring-for-user-defined-projects_preparing-to-configure-the-monitoring-stack-uwm). It's required for both sidecar mode and ambient mode metrics integration.
+
+```sh
+cat <<EOF | kubectl apply -f-
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    enableUserWorkload: true
+EOF
+
+kubectl wait --for=condition=Ready pods --all -n openshift-user-workload-monitoring --timeout 60s
```

Comment:
As this is being added to the "openshift-service-mesh" doc, it is better to use OpenShift commands like "oc". Otherwise, we would need to make a separate version of the procedure for OpenShift. This will also be consistent with the [ambient install instructions, which use OC](https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ambient/README.md#3-procedure-to-install-istios-ambient-mode).